app/partials/events.html

```diff
@@ -0,0 +1,24 @@
+<div cg-busy="{promise: loadingPromise, templateUrl:'partials/loadingtemplate.html'}">
+<span ng-if="events">
+    <section id="events">
+        <div ng-if="events.length == 1">
+            <h2>{{events[0].attributes.title}}</h2>
+            <p>{{events[0].attributes.description}}</p>
+            <a role="button" href="/checkin/{{events[0].id}}" class="button flatButton" ui-sref="checkin({id: events[0].id })">Check in to this Event</a>
+        </div>
+        <table ng-if="events.length > 1" ng-repeat="event in events">
+            <tr>
+                <th><p>{{event.attributes.title}}</p></th>
+                <td><a role="button" href="/checkin/{{event.id}}" class="button flatButton" ui-sref="checkin({id: events[0].id })">Check in to this Event</a></td>
```

Comment:
You don't need to specify both `href` and `ui-sref` attributes here. UI router will generate the `href` attribute